aws-ebs/kubernetes/latest/controller.yaml

```diff
@@ -20,6 +20,9 @@ rules:
   - apiGroups: ["storage.k8s.io"]
     resources: ["storageclasses"]
     verbs: ["get", "list", "watch"]
+  - apiGroups: ["storage.k8s.io"]
+    resources: ["csinodes"]
+    verbs: ["get", "list", "watch"]
```

Comment:
Upstream also added `node` resource https://github.com/kubernetes-sigs/aws-ebs-csi-driver/commit/22d307b024ebf263f05d3fab72e42ae8d2c8873d#diff-97777ab7d2c4944818065195f23faf0aR30

Maybe we should do the same?

Reply:
Umm, I don't see why not. We should add it

Reply:
Weird... I didn't catch any errors.